libs/rapidsandrivers-extras/src/main/kotlin/rapidsandrivers/EventNames.kt

```diff
@@ -6,6 +6,7 @@ enum class EventNames(val eventname: String) : EventnameHendelseType {
     GRUNNLAGSVERSJONERING_EVENT_NAME("GRUNNLAGSVERSJONERING_EVENT_NAME"),
     FEILA("FEILA"),
     FORDELER_STATISTIKK("FORDELER:STATISTIKK"),
+    ALDERSOVERGANG("ALDERSOVERGANG"),
```

Comment:
Bør nok være generisk (a la appnavnet) slik at man ikke trenger flere lyttere f.eks. for aldersovergang, og noe annet for revurdering oms pga x måneder siden dødsfall.